src/platforms/electron/devServer.js

```diff
@@ -1,18 +1,13 @@
-const fs = require('fs');
-const path = require('path');
 const chalk = require('chalk');
 const webpack = require('webpack');
 const merge = require('webpack-merge');
 const middleware = require('webpack-dev-middleware');
 const express = require('express');
 const app = express();
 
-// TODO: Spawn separate threads so realtime status logging can be used
-// without overwriting information/execptions logged by the compilers
-const logRealtime = str => {
-  let lineCount = (str.match(/\n/) || []).length + 1;
-  console.log('\u001B[' + lineCount + 'F\u001B[G\u001B[2K' + str);
-};
+// Ideally this would come from the same source that sets this port in webpack.web.config.js
+// but we can't inject a variable into this file because it is not being touched by webpack
+const WEBPACK_PORT = 9090;
```

Comment:
It looks like 9090 appears in several places in the code. Since we have to DRY break for pragmatic reasons here, the best way to handle this is:

- Note in a comment by the primary definition (presumably webpack) where all secondary usages occur
- Reference that devs should look at the primary definition in all secondary usages